addons/l10n_cl/views/res_partner.xml

```diff
@@ -15,8 +15,11 @@
             <field name="state_id" position="attributes">
                 <attribute name="placeholder">Región</attribute>
             </field>
-            <field name="country_id" position="after">
-                <field name="l10n_cl_sii_taxpayer_type"/>
+            <field name="vat" position="before">
+                <label for="l10n_cl_sii_taxpayer_type" string="Tax Payer Type"/>
```

Comment:
Is this <label for=... really needed?  If it is not in the address or in a special nolabel div, it should put the label automatically.

Reply:
Yes @jco-odoo. This is without label:


<img width="540" alt="Captura de pantalla 2019-11-14 20 42 46" src="https://user-images.githubusercontent.com/382893/68905261-559db180-071f-11ea-9250-bb8cb5abd4e1.png">



and this is with label:

<img width="455" alt="Captura de pantalla 2019-11-14 20 40 45" src="https://user-images.githubusercontent.com/382893/68905179-0f485280-071f-11ea-9993-90939a03b883.png">

The xpath location is positioned before vat field.

Reply:
Then I prefer to do it like in Argentina.  (have the same logic): 
```

            <field name="vat" position="after">
                <field name="l10n_ar_afip_responsibility_type_id" string="AFIP Responsibility" options="{'no_open': True, 'no_create': True}" attrs="{'readonly': [('parent_id','!=',False)]}"/>
            </field>
```

Reply:
Also the readonly parent_id != False should be added